Source/WebCore/animation/AnimationTimelinesController.h

```diff
@@ -115,7 +115,8 @@ class AnimationTimelinesController final : public CanMakeCheckedPtr<AnimationTim
     UncheckedKeyHashMap<FramesPerSecond, ReducedResolutionSeconds> m_animationFrameRateToLastTickTimeMap;
     WeakHashSet<AnimationTimeline> m_timelines;
     TaskCancellationGroup m_currentTimeClearingTaskCancellationGroup;
-    Document& m_document;
+    WeakRef<Document, WeakPtrImplWithEventTargetData> m_document;
+    Ref<Document> protectedDocument() const { return m_document.get(); }
```

Comment:
Methods are typically defined prior to instance variables, could you move this up a bit?

Reply:
Sure, thanks for the review.